keyboards/system76/launch_1/rules.mk

```diff
@@ -1,33 +1,23 @@
-# MCU name
-MCU = atmega32u4
+# AVR MCU options
+MCU = atmega32u4# MCU name
+BOOTLOADER = atmel-dfu# Bootloader selection
+F_CPU = 8000000# CPU frequency divided by two since AVR is at 3.3 V
+F_USB = 16000000# External oscillator is 16 MHz
 
-# CPU frequency divided by two since AVR is at 3.3 V
-F_CPU = 8000000
-
-# External oscillator is 16 MHz
-F_USB = 16000000
-
-# Bootloader selection
-BOOTLOADER = atmel-dfu
-
-# Build options
-#   change yes to no to disable
-BOOTMAGIC_ENABLE = no       # Bootmagic Lite
+# Feature options
```

Comment:
```suggestion
# Build options
#   change yes to no to disable
#
```

Reply:
I get you want it to be exactly the same on all of the "rules" Makefiles. If you want to enforce this, may I recommend you do so in an automated fashion.